"This API is deprecated." When fetching a db with no id.

When fetching a Database with an empty ID I get a misleading error "This API is deprecated.". 

```go
	client := notionapi.NewClient(notionapi.Token("mytoken"))

	db, err := client.Database.Get(context.TODO(), notionapi.DatabaseID(""))
	if err != nil {
		panic(err.Error())
	}
```

```sh
panic: This API is deprecated.
```

--- a/database.go
+++ b/database.go
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/pkg/errors"
 	"log"
 	"net/http"
 	"time"
@@ -117,6 +118,10 @@ type DatabaseQueryRequest struct {
 
 // See https://developers.notion.com/reference/get-database
 func (dc *DatabaseClient) Get(ctx context.Context, id DatabaseID) (*Database, error) {
+	if id == "" {
+		return nil, errors.New("empty database id")
+	}
+
 	res, err := dc.apiClient.request(ctx, http.MethodGet, fmt.Sprintf("databases/%s", id.String()), nil, nil)
 	if err != nil {
 		return nil, err

--- a/database_test.go
+++ b/database_test.go
@@ -439,6 +439,14 @@ func TestDatabaseClient(t *testing.T) {
 			})
 		}
 	})
+
+	t.Run("Get with empty database_id", func(t *testing.T) {
+		client := notionapi.NewClient("some_token")
+		_, err := client.Database.Get(context.TODO(), notionapi.DatabaseID(""))
+		if err.Error() != "empty database id" {
+			t.Error("database id is required error is expected")
+		}
+	})
 }
 
 func TestDatabaseQueryRequest_MarshalJSON(t *testing.T) {

--- a/http/examples.http
+++ b/http/examples.http
@@ -9,6 +9,7 @@ Notion-Version: {{api_version}}
 #Get
 GET https://api.notion.com/v1/databases/{{database_id}}
 Authorization: Bearer {{token}}
+Notion-Version: {{api_version}}
 
 ###
 # Query

